### DISCUSSION TITLE
How to get chat_id from a t.me/+invitelink ?

### DISCUSSION THREAD

**Discussion by d3xt3r01** [eternnoir/pyTelegramBotAPI#1842] [Q&A]
1. What version of pyTelegramBotAPI are you using?
4.8.0

2. What OS are you using?
gentoo

3. What version of python are you using?
3.10.9

Is there a way to get the chat_id (title too maybe? type?) from a t.me/+invitelink ?

**Comment by coder2020official** [comment]
I don't think it is possible through bot api, but it might be possible by using mtproto directly.

**Comment by d3xt3r01** [comment]
:-( Ok well then... I'm lost :))
I was thinking that maybe get_chat could support that but it seems it doesn't...

**Comment by Badiboy** [comment]
I suppose invite link is created especially that it was not able to understand to whom it belongs. :)

Anyway bot cannot get anything about the chat (channel, group, user etc.) where it was not added/started.

**Comment by d3xt3r01** [comment]
I beg to differ. It can get informations about channels/groups/supergroups it's not in using the get_chat() function. Tested and working.
That's why I was searching for something similar for the invite links.

**Comment by d3xt3r01** [comment]
Yes, it seems pyrogram can take a `t.me/+something` as a param and resolve it for both users and bots.
